eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -31,23 +33,23 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="PublishTrimmed" DependsOnTargets="RestoreProject;UpdateRuntimePack;Publish" />
+  <Target Name="PublishTrimmed" DependsOnTargets="ProcessFrameworkReferences;RestoreProject;UpdateRuntimePack;Publish" />
 
   <ItemGroup>
     <KnownFrameworkReference Include="Microsoft.NETCore.App"
```

Comment:
`KnownFrameworkReference` and `KnownAppHostPack` items can go away once we move to a `6.0` SDK, right? This is just to workaround the state we are in right now: targeting a `net6.0` app, but we are using a `5.0` SDK.

Reply:
Correct but we will hit the same issue when we update to net7.0 and use a NET 6.0 SDK. A sustainable solution would be ideal.

Reply:
I would imagine leaving this here and we can condition it by maybe parsing the TFM and the version in global.json and if they don't match then do this? How does that sound?